notebooks/workflow_cost_estimator/main.py

```diff
@@ -77,9 +80,13 @@ def estimate_job_cost(cpus: int, memory_gb: int, runtime_hours: float, preemptib
 
 with herzog.Cell("python"):
     report.style.format(dict(cost="${:.2f}", duration="{:.2f}h", memory="{:.0f}GB"))
+    # If the output here is blank, your submission probably hasn't started yet. Try re-running this notebook in a few minutes.
 
 with herzog.Cell("python"):
-    print("Total cost: $%.2f" % report['cost'].sum())
+    try:
+        print("Total cost: $%.2f" % report['cost'].sum())
+    except KeyError:
+        print("Cannot find cost in report dataframe -- your submission likely hasn't started execution.")
```

Comment:
Please revert this block: Instead, we should figure out how to omit incomplete lines from the cost report. This can be handled in a future PR.